char sensor_color
char fms_color
---
float32 rotate
